### DISCUSSION TITLE
👎  Font not working when run ``Next build``: Always have a class override my font on Next build

### DISCUSSION THREAD

**Discussion by AndreinNguyen** [vercel/next.js#48264] [Help] [answered]
### Summary

Every things work in dev server but always has a class override my font when run ``next build``

<img width="625" alt="image" src="https://user-images.githubusercontent.com/107632787/231261569-4c1d01f8-3706-421a-8a85-ef85f579ef13.png">

I import my font here in pages/_app.tsx
```
const irishGrover = Irish_Grover({ weight: "400", subsets: ["latin"] });
```

```
<main className={inriaSans.className}>
  <div>
    <Component {...pageProps} />
  </div>
</main>
```

Could I missing any thing?

**Comment by icyJoseph** [comment]
While it is not ideal that the dev server CSS ordering differs from the build, the issue here is plain CSS, aka, property-value combo that's defined last wins.

Which CSS framework are you using?

**Reply by icyJoseph** [reply]
@AndreinNguyen I am a bit short on time, could you try this:

```js
<main style={{fontFamily: inriaSans.style.fontFamily}}>
  <div>
    <Component {...pageProps} />
  </div>
</main>
```

Does it do the trick?

**Reply by AndreinNguyen** [reply]
Thank you very much 👏. It works !!!

**Reply by icyJoseph** [reply, marked as answer]
Nice! Please don't forget to mark as answered 🎉

**Reply by AndreinNguyen** [reply]
Thanks sir!

**Comment by AndreinNguyen** [comment]
Thanks for your response. I'm using MUI for CSS
